src/AppBundle/Repository/EventRepository.php

```diff
@@ -130,11 +130,9 @@ private function createUpcomingEventsQueryBuilder(): QueryBuilder
         return $qb
             ->leftJoin('e.committee', 'c')
             ->andWhere($qb->expr()->in('e.status', Event::ACTIVE_STATUSES))
-            ->andWhere('c.status = :committee_status')
```

Comment:
I don't understand why because we want to fetch events that are related to active committees.